libnetwork/drivers/bridge/setup_ip_tables_linux.go

```diff
@@ -67,9 +67,9 @@ func setupIPChains(config configuration, version iptables.IPVersion) (natChain *
 		}
 	}()
 
-	filterChain, err = iptable.NewChain(DockerChain, iptables.Filter)
+	_, err = iptable.NewChain(DockerChain, iptables.Filter)
 	if err != nil {
-		return nil, nil, nil, nil, fmt.Errorf("failed to create FILTER chain %s: %v", DockerChain, err)
+		return fmt.Errorf("failed to create FILTER chain %s: %v", DockerChain, err)
 	}
 	defer func() {
 		if err != nil {
```

Comment:
Just noticed this one; is this defer checking the wrong error? (should this be `retErr` ?)

If so, perhaps something we should fix separately before this one